meta/meta.go

```diff
@@ -985,29 +997,34 @@ func (m *Meta) UpdateDDLReorgStartHandle(job *model.Job, element *Element, start
 
 // UpdateDDLReorgHandle saves the job reorganization latest processed information for later resuming.
 func (m *Meta) UpdateDDLReorgHandle(job *model.Job, startKey, endKey kv.Key, physicalTableID int64, element *Element) error {
+	m.SetDiskFullOpt(kvrpcpb.DiskFullOpt_AllowedOnAlmostFull)
 	err := m.txn.HSet(mDDLJobReorgKey, m.reorgJobCurrentElement(job.ID), element.EncodeElement())
 	if err != nil {
 		return errors.Trace(err)
 	}
 	if startKey != nil {
+		m.SetDiskFullOpt(kvrpcpb.DiskFullOpt_AllowedOnAlmostFull)
```

Comment:
we need need to set all these place every time we put a key, we only need to set it in the initial phase of Meta(newMeta)

Reply:
below is the all the same

Reply:
This mark is associated with a specific operation instance and will be automatically cleaned up at the end of the transaction. Therefore, each operation needs to be marked again when it is executed

Reply:
Cleared